tips/TIP-0042/tip-0042.md

```diff
@@ -0,0 +1,1210 @@
+Since a staker has significant control over the rewards of the validator pool, formed by itself and its delegators, this
+unbonding period is in place to protect delegators from stakers changing the rules without prior notice. Once a staker
+goes into its unbonding period, delegators can be notified that a change is upcoming. Once an account has a Staking
+Feature whose `End Epoch` is the current one or has passed, it is no longer considered for validator selection and the
+Feature can be removed or updated. In the same transaction, the staker may claim their Mana Rewards for all epochs in
+which they were selected to participate in the validator committee.
+
+<details>
+  <summary>Staking Feature</summary>
+  <blockquote>Stakes IOTA coins to become eligible for committee selection, validate the network and receive Mana rewards.</blockquote>
+</details>
+<table>
+  <tr>
+    <td>
+      <b>Name</b>
+    </td>
+    <td>
+      <b>Type</b>
+    </td>
+    <td>
+      <b>Description</b>
+    </td>
+  </tr>
+  <tr>
+    <td>Feature Type</td>
+    <td>uint8</td>
+    <td>Set to <strong>value 7</strong> to denote a <i>Staking Feature</i>.</td>
+  </tr>
+  <tr>
+    <td>Staked Amount</td>
+    <td>uint64</td>
+    <td>The amount of IOTA coins that are locked and staked in the containing account.</td>
+  </tr>
+  <tr>
+    <td>Fixed Cost</td>
+    <td>uint64</td>
+    <td>The fixed cost of the validator, which it receives as part of its Mana rewards.</td>
+  </tr>
+  <tr>
+    <td>Start Epoch</td>
+    <td>uint64</td>
+    <td>The epoch index in which the staking started.</td>
+  </tr>
+  <tr>
+    <td>End Epoch</td>
+    <td>uint64</td>
+    <td>The epoch index in which the staking ends.</td>
+  </tr>
+</table>
+
+### Additional Transaction Syntactic Validation Rules
+
+- The `Amount` in the containing account output satisfies `Amount >= Staked Amount`.
+
+### Additional Transaction Semantic Validation Rules
+
+- Let `Future Bounded Epoch Index` be the epoch index corresponding to the slot index given by
+  `Commitment Index + Min Committable Age` where `Commitment Index` is the slot index of the commitment input.
+- Let `Past Bounded Epoch Index` be the epoch index corresponding to the slot index given by
+  `Commitment Index + Max Committable Age` where `Commitment Index` is the slot index of the commitment input.
+- When a _Staking Feature_ is present in an account output on the input or output side of a transaction, the transaction
+  that contains this output is valid, if and only if:
+  - A _Commitment Input_ is present.
+- When a _Staking Feature_ is present in an account output on the output side of a transaction, the transaction that
+  contains this output is valid, if and only if all of the following conditions hold:
+  - A _Block Issuer Feature_ must be present in the account.
+- When a _Staking Feature_ is present in an account output on the output side of a transaction and it was not present on
```

Comment:
What was not present? The output or the feature?

Reply:
Or both :)

Reply:
Clarified in the TIP :)